logger/interfaces/interfaces.go

```diff
@@ -29,4 +31,6 @@ type Logger interface {
 	WithFields(fields map[string]interface{}) Logger
 	WithField(key string, value interface{}) Logger
 	WithError(err error) Logger
+
+	LogrusLogger() logrus.FieldLogger
```

Comment:
Can you explain the need for this method? It sounds a bit weird that we are tying our generic Logger interface to Logrus

Reply:
So in a few places I need to get the internal logger that this interface uses, how about this?

`GetInternalLogger() any`

Reply:
Can't we define the internal logger as another interface rather than using any?

Reply:
Like this `GetInternalLogger() logrus.FieldLogger`

Reply:
no because this interface is too extensive and use internal logrus stuff